internal/build/docker.go

```diff
@@ -43,6 +43,12 @@ type Cmd struct {
 	Argv []string
 }
 
+// entrypointStr returns the Cmd in string form for use as an entrypoint,
+// e.g. 'ENTRYPOINT ["executable", "param1", "param2"]'
+func (c Cmd) entrypointStr() string {
```

Comment:
should this be a pointer receiver?

Reply:
why should it be a pointer receiver? we mostly don't pass cmd's around as pointers, and I'm loath to start b/c it'd just add extra `nil` checks. (i am passing the cmd as a pointer to `startContainer` but that's just b/c it's optional.)